### PR TITLE
Create Code of Conduct

### PR DESCRIPTION
**Pull request by dschaper**
Initial copy of our Discourse Code of Conduct.

Signed-off-by: Dan Schaper <dan.schaper@pi-hole.net>

### CODE REVIEW

**Comment by dschaper** [comment]
@pi-hole RFC

**Comment by PromoFaux** [comment]
Looks good to me, this is pulled from discourse, right?

**Comment by dschaper** [comment]
Yes, it's a direct copy. Not sure if the text about being a discussion/forum is proper for GitHub?

**Comment by diginc** [comment]
Sounds fine to me, should it be cited as discourse's at the end of the text?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Instead, provide reasoned counter-arguments that improve the conversation.
+
+<a name="participate"></a>
+
+## [Your Participation Counts](#participate)
+
+The conversations we have here set the tone for everyone. Help us influence the future of this community by choosing to engage in discussions that make this forum an interesting place to be &mdash; and avoiding those that do not.
+
+Discourse provides tools that enable the community to collectively identify the best (and worst) contributions: favorites, bookmarks, likes, flags, replies, edits, and so forth. Use these tools to improve your own experience, and everyone else’s, too.
```
Comment (diginc) [inline CODE_OF_CONDUCT.md]: Discourse specific

Comment (dschaper) [reply CODE_OF_CONDUCT.md]: Yeah, it needs some modifications and edits.

**Comment by dschaper** [comment]
https://raw.githubusercontent.com/discourse/discourse/master/docs/code-of-conduct.md better option?

**Comment by dschaper** [comment]
See #4 for options.

**Comment by dschaper** [comment]
Closing in favor of #4

**Pull request closed without merging**